connect/src/com/telenor/connect/ConnectSdk.java

```diff
@@ -106,7 +105,7 @@ public static synchronized void authenticate(
             final Map<String, String> parameters,
             final int requestCode) {
         Validator.sdkInitialized();
-        sdkProfile.onStartAuthorization(parameters, new SdkProfile.OnStartAuthorizationCallback() {
+        sdkProfile.onStartAuthorization(parameters, new SdkProfile.OnStartAuthenticationCallback() {
```

Comment:
I'm not sure about that renaming. Don't we want to use authorization everywhere instead of authentication.
@phyrex1an ?
However it's better that we are not consistent at least.